src/tuple.alp

```diff
@@ -0,0 +1,82 @@
+{-| Some helpers for working with 2-tuples.
```

Comment:
Out of curiousity, I'm not familiar with the `{-|` comment syntax.  Where is it from?

Reply:
Ah! It's the syntax for a documentation comment in haskell and elm, meaning that the comment will be used when generating pretty html documentation for the module. I wrote it habitually.

Shall I continue to use this out shall I remove the pipe to make it a normal comment?

Reply:
Short answer:
Doesn't bother me to leave it in atm but let's discuss it further under Alpaca :)

Longer answer:
I'm vaguely of the opinion that any comment preceding an AST node should be automatically attached to that node as documentation for output so I'm not sure if we need a special form (this ties into formatters and alternate ways of storing source).  But!  I think this is a good thing to discuss, especially here:  https://github.com/alpaca-lang/alpaca/issues/134